tests/unit/src/test/scala/tests/codeactions/RewriteBracesParensLspSuite.scala

```diff
@@ -52,6 +53,7 @@ class RewriteBracesParensLspSuite
        |}
        |""".stripMargin,
     s"""|${RewriteBracesParensCodeAction.toBraces("foreach")}
+        |${ExtractMethodCodeAction.title("object `Main`")}
```

Comment:
Shouldn't we show the new code action only if a range is selected? Otherwise it's hard to figure out what to extract.